### PR TITLE
Add IsFuel Condition

### PR DESCRIPTION
**Pull request by Mr-Darth**
### Description
Added a condition to check whether an itemtype can be used as furnace fuel.

---
**Target Minecraft Versions:** 1.11+
**Requirements:** 1.11+
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/conditions/CondIsFuel.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import ch.njol.skript.doc.Since;
+
+@Name("Is Fuel")
+@Description("Checks whether an item can be used as fuel in a furnace.")
+@Examples({"on right click on furnace:",
+	"\tif player's tool is not fuel:",
+	"\t\tsend \"Please hold a valid fuel item in your hand\"",
+	"\t\tcancel event"
+})
+@Since("INSERT VERSION")
```
Comment (WeeskyBDW) [inline src/main/java/ch/njol/skript/conditions/CondIsFuel.java:40]: Set it to 2.5.1 i think

**Review by WeeskyBDW — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/conditions/CondIsFuel.java (hunk 1 above)
Comment (Romitou) [reply src/main/java/ch/njol/skript/conditions/CondIsFuel.java:40]: The 'INSERT VERSION' mentions will be replaced at the time of release.

File: src/main/java/ch/njol/skript/conditions/CondIsFuel.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@RequiredPlugins("1.11+")
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/conditions/CondIsFuel.java:41]: Looks like this was added in 1.11.2
Should be changed just for extra clarity (even though Skript doesn't officially any other 1.11.x version)
```suggestion
@RequiredPlugins("1.11.2+")
```

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

**Review by APickledWalrus — APPROVED** [review]
LGTM

File: src/main/java/ch/njol/skript/conditions/CondIsFuel.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+@Name("Is Fuel")
+@Description("Checks whether an item can be used as fuel in a furnace.")
+@Examples({"on right click on furnace:",
+	"\tif player's tool is not fuel:",
+	"\t\tsend \"Please hold a valid fuel item in your hand\"",
+	"\t\tcancel event"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("1.11.2+")
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsFuel.java:41]: To keep consistency with other classes, we should probably do "Minecraft 1.11.2+" here

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
Just one small tiny request, other than that, looks good to me.

**Review by ShaneBeee — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by ShaneBeee** [comment]
Thank you for your contribution

**Pull request merged**